### PR TITLE
[Fuzzer] add labels for runtime fuzzer

### PR DESCRIPTION
**Pull request by axelcisco**

### CODE REVIEW

File: charts/apiclarity/templates/fuzzer-template-configmap.yaml [hunk 1]
```diff
@@ -12,14 +12,14 @@ data:
     metadata:
       namespace: '{{ .Release.Namespace }}'
       labels:
-{{- toYaml (index .Values "apiclarity" "labels") | nindent 8 }}
+{{- toYaml (index .Values "APIClarityRuntimeFuzzing" "labels") | nindent 8 }}
```
Comment (FrimIdan) [inline charts/apiclarity/templates/fuzzer-template-configmap.yaml:15]: {{- toYaml .Values.APIClarityRuntimeFuzzing.labels | nindent 8 }}

File: charts/apiclarity/templates/fuzzer-template-configmap.yaml [hunk 2]
```diff
@@ -12,14 +12,14 @@ data:
...
     spec:
       backoffLimit: 0
       ttlSecondsAfterFinished: 300
       template:
         metadata:
           labels:
-{{- toYaml (index .Values "apiclarity" "labels") | nindent 12 }}
+{{- toYaml (index .Values "APIClarityRuntimeFuzzing" "labels") | nindent 12 }}
```
Comment (FrimIdan) [inline charts/apiclarity/templates/fuzzer-template-configmap.yaml:22]: {{- toYaml .Values.APIClarityRuntimeFuzzing.labels | nindent 12 }}

**Review by FrimIdan — APPROVED** [review]

File: charts/apiclarity/templates/fuzzer-template-configmap.yaml (hunk 1 above)
Comment (axelcisco) [reply charts/apiclarity/templates/fuzzer-template-configmap.yaml]: done

File: charts/apiclarity/templates/fuzzer-template-configmap.yaml (hunk 2 above)
Comment (axelcisco) [reply charts/apiclarity/templates/fuzzer-template-configmap.yaml]: done (+resources)

**Pull request merged**